src/main/java/seedu/duke/classes/Show.java

```diff
@@ -12,13 +12,15 @@
     protected LocalDateTime showTime;   //The time of the show, maybe include date
     protected int currentSeason;    //to keep track of watch progress
     protected int currentEpisode;
+    protected int showDuration; //duration of the show in minutes
 
-    public Show(String name, int numSeasons, int[] numEpisodesForSeasons) {
+    public Show(String name, int showDuration, int numSeasons, int[] numEpisodesForSeasons) {
```

Comment:
What if we don't know showDuration?

Reply:
Just assume they know or set default.

Reply:
Just assume they know. Dont set default uh